opentelemetry-userevents-exporter/examples/basic.rs

```diff
@@ -0,0 +1,55 @@
+#![allow(unused_imports, unused_mut, unused_variables, dead_code)]
+
+//! run with `$ cargo run --example basic --all-features
+
+use opentelemetry_api::{
+    logs::LoggerProvider as _,
+    logs::LogRecordBuilder,
+    logs::Logger,
+    logs::Severity,
+    trace::{Span, Tracer, TracerProvider as _, mark_span_as_active},
+    Context, KeyValue, logs::AnyValue
+};
+
+use opentelemetry_sdk::{
+    logs::LoggerProvider,
+    runtime,
+    trace::TracerProvider,
+};
+
+use opentelemetry_userevents_exporter_log::{ExporterConfig, RealTimeLogProcessor, ProviderGroup};
+use std::time::SystemTime;
+use std::thread;
+
+fn init_logger() -> LoggerProvider {
+    let exporter_config = ExporterConfig{keyword : 1};
+    let prov_group = Some("prov_group");
+    let realtime_processor = RealTimeLogProcessor::new("test12345", None , exporter_config);
+    LoggerProvider::builder()
+    .with_log_processor(realtime_processor).build()
+}
+
+fn main() {
+    let logger_provider = init_logger();
+    let prov_group = Some("test1234");
+    let logger: opentelemetry_sdk::logs::Logger = logger_provider.logger("test");
```

Comment:
lets use `log` crate to do the logging, so this example can use the log-appender. There is pretty good chance this example will misled users to believe *this is the way* to log in OTel Rust!

Reply:
Good point, as of now I have added log-appender example. But it is very limited to export unstructured text. Have kept log-bridge example with disclaimer just as it helps to test the changes better. We can remove it once tracing subscriber example is added. But I don't have any strong opinion - we can remove it if it's really causing confusion.